### DISCUSSION TITLE
just curious about setups where volume control does work?

### DISCUSSION THREAD

**Discussion by jmreeves11** [MonitorControl/MonitorControl#1239] [Q&A]
I, like other users, have a MacBook Pro connected to a Dell monitor and Dell sound-output volume-control from mac's buttons does not work. MonitorControl does have the volume-controlling feature, though... so, what types of setups (any particular devices?) are there where that volume control does work?

**Comment by white-hat** [comment]
sound controls work with my LG 43UN700 and intel macbook pro 16"
and only thing I changed in settings - "Screen to control" is set to "use audio device name"
